docs/examples/800_istio.adoc

```diff
@@ -0,0 +1,169 @@
+
+[source,bash]
+----
+kubectl create namespace coherence-cluster
+kubectl label namespace coherence-cluster istio-injection=enabled
+----
+
+There is no other requirements to run Coherence in Istio environment.
+
+The following is an example that creates a cluster named example-cluster:
+
+example.yaml
+[source,bash]
+----
+# Example
+apiVersion: coherence.oracle.com/v1
+kind: Coherence
+metadata:
+  name: example-cluster
+----
+
+[source,bash]
+----
+$ kubectl -n coherence-cluster apply -f example.yaml
+----
+
+After you installed the Coherence cluster, run the following command to view the pods:
+
+[source,bash]
+----
+$ kubectl -n coherence-cluster get pods
+
+NAME                                             READY   STATUS    RESTARTS   AGE
+example-cluster-0                                2/2     Running   0          45m
+example-cluster-1                                2/2     Running   0          45m
+example-cluster-2                                2/2     Running   0          45m
+----
+
+You can see that 3 members in the cluster are running with 3 pods. 2/2 in READY column means that there are 2 containers running in each Pod. One is Coherence member and the other is Envoy Proxy.
+
+== Prometheus
+
+You can use Prometheus with Istio to record metrics that track the health of Istio and of Coherence cluster within the service mesh and view them through Granfana. 
+
+== Traffic Management
+
+Istio provides traffic management capabilities, including the ability to visualize traffic in Kiali. You do not need to change your applications to use this feature. The Istio proxy (envoy) sidecar that is injected into your pods provides it. The image below shows an example with traffic flowing: In from the Istio gateway on the left, to a cluster called example-cluster in the coherence-example namespace.
+
+In this example, you can see how the traffic flows to the cluster services and then to the individual cluster members.  This example has storage members (example-cluster-storage), a member running proxy service (example-cluster-proxy), and a member running http server (example-cluster-rest).  
+
+image::../images/istioKiali.png[width=1024,height=512]
+
+To learn more, see https://istio.io/latest/docs/concepts/traffic-management/[Istio traffic management].
+
+== Distributed Tracing
+
+Istio provides distributed tracing capabilities, including the ability to view traces in Jaeger. In order to use distributed tracing, you will need to enable Coherence Distributed Tracing. For example:
+
+example.yaml
+[source,bash]
+----
+# Example
+apiVersion: coherence.oracle.com/v1
+kind: Coherence
+metadata:
+  name: example-cluster
+spec:
+  coherence:
+    tracing:
+      ratio: 1
+----
+
+The images below show an example of a distributed trace flow and details of a trace span.
+
```

Comment:
Wouldn't these example images be more interesting if they showed a proper Coherence request which has multiple spans rather than something simple like REST? For example a request from a Coherence Extend client to a multi-member storage cluster doing a "put" that so you can see how Istio adds to the tracing that Coherence already provides (if Istio does add anything)

Reply:
Decided to remove open tracing/Jaeger examples since Istio traces couldn't show flow between Coherence members.  But traces from Istio and Coherence can been viewed independently.